### PR TITLE
CHAOS-3347 Alert integration for chaos experiments

### PR DESCRIPTION
**Pull request by SmritiSatya**
[Document alert integration for chaos experiment:How to create a channel, a rule and applying the rule.](https://667bfaf1f557792b3558d067--harness-developer.netlify.app/docs/chaos-engineering/features/experiments/alert-integration)

### CODE REVIEW

File: docs/chaos-engineering/features/experiments/alert-integration.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+---
+
+This topic describes how you can create email alerts for your chaos experiments.
+
+
+## What are alerts?
+Alerts are a form of communication that provide updates about your experiment runs. These alerts are provided in the form of email updates that describe your chaos experiments that started, stopped, and completed (depending on which type of experiments you configured to receive the alerts for).
+
+## Why are they required?
+Alerts provide the status of the chaos experiments and other details (such as run details, fault configurations, probes used, and so on) which may be required in making important business decisions. They can also be used as historical data of the earlier chaos experiment runs.
```
Comment (douglas-j-bothwell) [inline docs/chaos-engineering/features/experiments/alert-integration.md:14]:
```suggestion
Alerts provide the status of the chaos experiments and other details (such as run details, fault configurations, probes used, and so on) that might be required when making important business decisions. You can also use alerts as historical data for the earlier chaos experiment runs.
```

File: docs/chaos-engineering/features/experiments/alert-integration.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+## How to create and use alerts?
+You can create an alert by [creating a channel](#create-a-channel), and [creating conditions](#create-a-condition). You can [apply these conditions](#select-channel-for-alerts) to a rule.
```
Comment (douglas-j-bothwell) [inline docs/chaos-engineering/features/experiments/alert-integration.md:17]:
```suggestion
You can create an alert by [creating a channel](#create-a-channel) and [creating conditions](#create-a-condition). You can [apply these conditions](#select-channel-for-alerts) to a rule.
```

File: docs/chaos-engineering/features/experiments/alert-integration.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+:::note
+This feature is currently behind the feature flag, `PL_CENTRAL_NOTIFICATIONS`. Contact [Harness Support](mailto:support@harness.io) to enable this feature.
```
Comment (douglas-j-bothwell) [inline docs/chaos-engineering/features/experiments/alert-integration.md:20]:
```suggestion
This feature is currently behind the feature flag `PL_CENTRAL_NOTIFICATIONS`. Contact [Harness Support](mailto:support@harness.io) to enable this feature.
```

File: docs/chaos-engineering/features/experiments/alert-integration.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+:::
+
+:::tip
+- Ensure that you have SMTP configured on your account. HCE configures SMTP for you as a part of the basic setup for your application.
+- To configure SMTP, ensure your account is connected to one delegate (irrespective of the scope).
+:::
+
+### Ensure SMTP is configured
+
+1. To ensure SMTP is configured for your account, navigate to **Administrative Settings**.
```
Comment (douglas-j-bothwell) [inline docs/chaos-engineering/features/experiments/alert-integration.md:30]:
```suggestion
1. To ensure that SMTP is configured for your account, navigate to **Administrative Settings**.
```

File: docs/chaos-engineering/features/experiments/alert-integration.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+
+    ![](./static/alert-integration/setup-done-3.png)
+
+### Create a channel
+
+3. Navigate to **Chaos Engineering** module.
+
+    ![](./static/alert-integration/chaos-mod-4.png)
+
+4. Select **Project Settings** and select **Notifications Management**.
```
Comment (douglas-j-bothwell) [inline docs/chaos-engineering/features/experiments/alert-integration.md:44]:
```suggestion
4. Select **Project Settings** and then **Notifications Management**.
```

File: docs/chaos-engineering/features/experiments/alert-integration.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+
+    ![](./static/alert-integration/see-channel-8.png)
+
+:::tip
+Ensure you create a channel before creating a rule because a channel decides which method (email, slack, and so on) you wish to receive the alerts.
+:::
+
+### Create a rule
+
+7. To create a new rule, select **+ New Notification**.
```
Comment (douglas-j-bothwell) [inline docs/chaos-engineering/features/experiments/alert-integration.md:70]: I'm kinda thinking you should renumber the steps at the start of each section.

File: docs/chaos-engineering/features/experiments/alert-integration.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+
+    ![](./static/alert-integration/condition-11.png)
+
+### Create a condition
+
+11. Provide the **Condition Name**, events (such as chaos experiment started, stopped, or completed) for which you wish to receive the alerts. Select **Apply**.
+
+    ![](./static/alert-integration/create-12.png)
+
+12. You will see that some conditions have been configured for your rule. Select **Continue**.
```
Comment (douglas-j-bothwell) [inline docs/chaos-engineering/features/experiments/alert-integration.md:96]: Optional sug: Try to reword this using present tense.

File: docs/chaos-engineering/features/experiments/alert-integration.md [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+14. Select one of the channel/s that you configured earlier and select **Apply Selected**. To create a new channel, select **New Channel**.
+
+    ![](./static/alert-integration/apply-15.png)
+
+15. You can **Enable** or disable the rule based on whether you select or de-select the radio button. By default, it is **Enable on Save**. Select **Submit**.
+
+    ![](./static/alert-integration/submit-16.png)
+
+:::tip
+If you try to create two rules on the same chaos experiment, you can seamlessly do so, and this results in you receiving two (or the number of rules you created the alerts) alerts for the same chaos experiment.
```
Comment (douglas-j-bothwell) [inline docs/chaos-engineering/features/experiments/alert-integration.md:115]:
```suggestion
If you try to create multiple rules on the same chaos experiment, you can seamlessly do so, and this results in you receiving multiple alerts (one per rule) for the same chaos experiment.
```
Optional sug, not sure if I'm changing the meaning here....

File: docs/chaos-engineering/features/experiments/alert-integration.md [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+:::
+
+### Sample run report
+
+Once you set up your email to receive alerts and the feature flag is enabled, you will receive a run report as an attachment similar to the below report. This report will describe the run details, fault details, and so on.
```
Comment (douglas-j-bothwell) [inline docs/chaos-engineering/features/experiments/alert-integration.md:120]:
```suggestion
Once you set up your email to receive alerts and the feature flag is enabled, you should receive a run report as an attachment similar to the below report. This report will describe the run details, fault details, and so on.
```

**Review by douglas-j-bothwell — DISMISSED** [review]
A few sugs otherwise LGTM

File: docs/chaos-engineering/features/experiments/alert-integration.md [hunk 10]
```diff
@@ -0,0 +1,131 @@
...
+
+    ![](./static/alert-integration/submit-16.png)
+
+:::tip
+If you try to create multiple rules on the same chaos experiment, you can seamlessly do so, and this results in you receiving multiple alerts (one per rule) for the same chaos experiment.
+:::
+
+### Sample run report
+
+Once you set up your email to receive alerts and the feature flag is enabled, you should receive a run report as an attachment similar to the below report. This report will describe the run details, fault details, and so on.
```
Comment (vanshBhatia-A4k9) [inline docs/chaos-engineering/features/experiments/alert-integration.md:120]: We may want to clarify that this report will only come if they have configured the chaos experiment completed event as part of their condition creation, and at present it is not configurable (you will always get the report if you configure the completed event, there is no option to toggle if this report comes or not)

**Review by vanshBhatia-A4k9 — APPROVED** [review]
Minor feedbacks, LGTM. Thanks!

**Pull request merged**